packages/openapi-generator/README.md

```diff
@@ -1,39 +1,89 @@
-## OpenApi Client Generator
+## SAP Cloud SDK OpenAPI Client Generator (Beta)
 
-This is currently not release ready. excluded in `mail.ts`.
+Generate custom JavaScript/TypeScript clients for OpenAPI services.
+This generator is based on the official [OpenAPI generator](https://openapi-generator.tech/) and adds some additional code for convenience to better integrate with the SAP Cloud SDK.
 
-The generator wraps only the openapi-generator.
-This is a flexible tool using many commandline flags and templates for easy adjustment.
-<br>[Command line arguments](https://github.com/OpenAPITools/openapi-generator#3---usage)
-<br>[Repo For mustache files](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/typescript-axios)
+## Prerequisites
 
-In the wrapping `generator-cli` we currently only expose `inputDir` and `outputDir` to generate tyescript-axios sources from the openapi.json files.
+The official OpenAPI generator is Java based, therefore you need to have a Java runtime installed to use the SAP Cloud SDK OpenAPI generator.
 
+# Installation
+```sh-session
+$ npm install @sap-cloud-sdk/openapi-generator
+```
+# Usage (CLI)
```

Comment:
I wonder if this `usage` section makes sense. Or is ot auto generated? If so then it is fine. Note that the oclif support auto generation.

Reply:
If it is not generated I would either do the generation or say that with the `help` command you can get infos because you are stating a console output here.

Reply:
It is autogenerated 😊